### PR TITLE
Added Log.py file

### PR DESCRIPTION
**Pull request by anushkrishnav**
I have set up a basic log file 
issue #26

### CODE REVIEW

**Comment by gat786** [comment]
Looks good to me

File: src/modules/log.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import logging
+import discord
+
+def create_logger() :
+    """Create a Logger."""
+
+    logger = logging.getLogger("discord')
```
Comment (adnan007d) [inline src/modules/log.py:7]: Quote error

**Review by ricardoprins — APPROVED** [review]
looks good to me as well!

**Comment by ricardoprins** [comment]
it just needs squash/merging, and then that's it

**Comment by gat786** [comment]
@all-contributors add @anushkrishnav for code

**Pull request merged**